wolfcrypt/src/asn.c

```diff
@@ -325,7 +325,6 @@ struct tm* gmtime(const time_t* timer)
 }
 #endif /* WOLFSSL_GMTIME */
 
```

Comment:
The time code in asn.c should be removed, but I still see these functions here, so we have duplicates. You should remove lines asn.c:120 - 458 and 478 - 492 once you move the missing functions.

Reply:
I forgot to remove it, too.
I deleted those lines from asn.c.